web/src/components/shared/modals/SkipPreflightsModal.jsx

```diff
@@ -22,9 +22,9 @@ export default function SkipPreflightsModal(props) {
           <div className="u-marginTop--30 flex flex-column">
             <button type="button" className="btn blue primary" onClick={hideSkipModal}>Wait for Preflights to finish</button>
             {onForceDeployClick ?
-              <span className="u-fontSize--normal u-fontWeight--medium u-textDecoration--underline u-textColor--bodyCopy u-marginTop--15 u-cursor--pointer" onClick={() => onForceDeployClick(false)}>Ignore Preflights and deploy</span>
+              <span className="u-fontSize--normal u-fontWeight--medium u-textDecoration--underline u-textColor--bodyCopy u-marginTop--15 u-cursor--pointer" onClick={() => onForceDeployClick(true)}>Ignore Preflights and deploy</span>
```

Comment:
why was this changed to `true`? it looks like the `onForceDeployClick` takes a `continueWithFailedPreflights` and in this case they didn't fail but rather skipped, right?

Reply:
correct, thanks, updated